annotations/templates/annotations/relationtemplate_list.html

```diff
@@ -46,9 +91,21 @@
 
 <script>
 jQuery(document).ready(function($) {
+
     $(".clickable-row").click(function() {
         window.document.location = $(this).data("href");
     });
+
+    $('[data-delete-template]').tooltip();
+
+    $("a[id^='openTemplateDeleteModal']").click(function(e) {
```

Comment:
just give all buttons the same class and then use the class to identify buttons here.

Reply:
The buttons now are identified by class.